app/assets/javascripts/maps.coffee.erb

```diff
@@ -258,6 +265,13 @@ KS.controllerUrl = (controller) ->
     when 'issues' then '<%= Rails.application.routes.url_helpers.issues_path %>'
     when 'jobs' then '<%= Rails.application.routes.url_helpers.jobs_path %>'
 
+KS.addMapSettingsToParam = ->
+  newUrl = window.location.href.replace(new RegExp('&center=[\\d\\.,]*', 'g'), '')
+    .replace(new RegExp('&zoom=[\\d\\.]*', 'g'), '')
+  newUrl = newUrl + '&center=' + KS.Map.getView().getCenter()
+  newUrl = newUrl + '&zoom=' + KS.Map.getView().getZoom()
+  history.pushState({}, null, newUrl);
```

Comment:
```suggestion
  newUrl = new URL(window.location.href)
  newUrl.searchParams.set 'center', KS.Map.getView().getCenter()
  newUrl.searchParams.set 'zoom', KS.Map.getView().getZoom()
  history.pushState {}, null, newUrl
```

Reply:
Die Parameter für `center` und `zoom` sollen doch in der Methode aktualisiert werden, richtig?

Reply:
Ja, Aktualisiert (falls bereits vorhanden) oder neu gesetzt.